### PR TITLE
Typo "a Microsoft Teams Room"→"Microsoft Teams Rooms"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/devices/check-in-and-room-release 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/devices/check-in-and-room-release.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/devices/check-in-and-room-release.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ When Teams panels are paired with a Teams Room on Android, a user can check in j
 ## Turn on check-in notifications
 
 > [!NOTE]
-> This feature is currently only available on Teams panels that are paired with a Teams Room on Android. The Teams panel and Teams room must be signed into the same resource account. See [Pair a Teams panel with a Microsoft Teams Room on Android](use-teams-panels.md#pair-a-teams-panel-with-a-microsoft-teams-room-on-android) to learn more.  
+> This feature is currently only available on Teams panels that are paired with a Teams Room on Android. The Teams panel and Teams room must be signed into the same resource account. See [Pair a Teams panel with Microsoft Teams Rooms on Android](use-teams-panels.md#pair-a-teams-panel-with-a-microsoft-teams-room-on-android) to learn more.
```
Comment (JohanFreelancer9) [inline Teams/devices/check-in-and-room-release.md:48]:
```suggestion
> This feature is currently only available on Teams panels that are paired with Teams Rooms on Android. The Teams panel and Teams Rooms must be signed into the same resource account. See [Pair a Teams panel with Microsoft Teams Rooms on Android](use-teams-panels.md#pair-a-teams-panel-with-a-microsoft-teams-room-on-android) to learn more.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the additional suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
close > reopen

File: Teams/devices/check-in-and-room-release.md (hunk 1 above)
Comment (tiburd) [reply Teams/devices/check-in-and-room-release.md:48]: In the article, the linked bookmark is for a section titled "Pair a Teams panel with a Microsoft Teams Room on Android":
image

**Comment by tiburd** [comment]
I'm closing this PR.
In the article, the linked bookmark is for a section titled "Pair a Teams panel with a Microsoft Teams Room on Android":
image

**Pull request closed without merging**